compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -3061,8 +3107,35 @@ class MultiLoweringConfigGenerator {
   llvm::SmallDenseMap<Operation *, SmallVector<int64_t>> nonRootOpVecTileSizes;
 };
 
-void MultiLoweringConfigGenerator::loadRootLoweringConfig(
-    IREE::CPU::LoweringConfigAttr rootLoweringConfig) {
+std::unique_ptr<MultiLoweringConfigGenerator>
+MultiLoweringConfigGenerator::create(Operation *rootOperation,
+                                     ArrayRef<Operation *> computeOps) {
+
```

Comment:
nit: delete the blank line

https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace